src/Components/Notifications/util.ts

```diff
@@ -14,3 +17,28 @@ const isArtworksBasedNotification = (
 ) => {
   return ["ARTWORK_ALERT", "ARTWORK_PUBLISHED"].includes(notificationType)
 }
+
+export const setLastSeenNotificationDate = date => {
+  if (!date) {
+    return
+  }
+
+  window.localStorage.setItem(LAST_SEEN_NOTIFICATION_PUBLISHED_AT_KEY, date)
+}
+
+export const hasNewNotifications = lastNotificationDatetime => {
+  if (!lastNotificationDatetime) {
+    return false
+  }
+
+  if (!window.localStorage.getItem(LAST_SEEN_NOTIFICATION_PUBLISHED_AT_KEY)) {
+    return true
+  }
+
+  return (
+    Date.parse(lastNotificationDatetime) >
```

Comment:
**suggestion**: use `DateTime.fromISO` (this approach is usually used in code) instead of `Date.parse`

Reply:
**suggestion**: some refactor suggestions

```javascript
export const hasNewNotifications = lastNotificationDatetime => {
  const prevLastNotificationDatetime = window.localStorage.getItem(LAST_SEEN_NOTIFICATION_PUBLISHED_AT_KEY)

  if (!lastNotificationDatetime) {
    return false
  }

  if (!prevLastNotificationDatetime) {
    return true
  }

  const prevDate = DateTime.fromISO(prevLastNotificationDatetime)
  const newDate = DateTime.fromISO(lastNotificationDatetime)

  return newDate > prevDate
}
```